srcpkgs/boringtun/template

```diff
@@ -0,0 +1,11 @@
+# Template file for 'boringtun'
+pkgname=boringtun
+version=vTEST2
+revision=1
+build_style=cargo
+short_desc="BoringTun is an implementation of the WireGuard® protocol designed for portability and speed."
```

Comment:
That description seems unnecessarily long to me

Reply:
`Implementation of the WireGuard protocol designed for portability and speed`

Reply:
also has a trailing dot

Reply:
Fixed, thanks!